### PR TITLE
Feature#1 setup project

### LINKED ISSUE #1: Setup project
# REQUIREMENTS
verify that they are in the corresponding modules as indicated by the gemfile.
    
Version of Ruby: 2.7
    
Version of Rails: 6.1.3
    
The gems for develop are: 
cancan https://github.com/CanCanCommunity/cancancan
pundit https://github.com/varvet/pundit 
devise 
will paginate
faker
figaro
caxlsx_rails
JWT https://github.com/jwt/ruby-jwt/ 
    
The gems for testing are:
factory_bot_rails

# DESIGN
Does not apply

# Acceptance Criteria
- [x] Created project with ruby on rails
- [x] Add all necessary gems for developing (back and front)
- [x] Add all necessary gems for testing
- [x] Add all necessary gems for deployment

### PR DESCRIPTION
**Pull request by mariormt17**
# Description
What did you implemented/Why did you implemented this:
 - Explain the changes you’ve made. 
 - It doesn’t need to be fancy and you don’t have to get to technical, yet. 
 - Just explicit what you have implemented at a high level. 
 - Let the reviewer know the overall effect of the PR.
 - Reference a ticket in your issue tracker. Explain what the change is and then and only then reference the ticket.
 - The “why” is a chance to explain both the engineering goal, but also a some business objective that is satisfied or moved along.

This is the project setup to start the other issues with the necessary gems

 Examples:
 - What?: I've added support for authentication to implement Key Result 2 of OKR1. It includes 
model, table, controller and test. For more background, see ticket #JIRA-123.
 - Why?: These changes complete the user login and account creation experience. See #JIRA-123 for more information.
 
 # Testing
 Help me how can I test or look at the changes
 
 Example:
  - I've added coverage for testing all new methods. I used Faker for a few random user emails and names.
  
  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes

### CODE REVIEW

File: alerta-covid/Gemfile [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# gem 'redis', '~> 4.0'
+# Use Active Model has_secure_password
+# gem 'bcrypt', '~> 3.1.7'
+
+gem 'cancancan'
+gem 'pundit'
+gem 'devise'
+gem 'will_paginate', '~> 3.1.0'
+gem 'faker', :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'
+gem 'figaro'
```
Comment (EduardoGHdez) [inline alerta-covid/Gemfile:30]: Just curious. 
Is this gem really needed?

How do you plan to use it?

Sorry, it's just that I didn't see it before
I think we can make what it offers with vanilla ruby/rails

File: alerta-covid/Gemfile [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+gem 'caxlsx'
```
Comment (EduardoGHdez) [inline alerta-covid/Gemfile:31]: Do we have any plans for generating xls/reports or something?

If this is not really neede, can we remove it?

**Review by EduardoGHdez — APPROVED** [review]
LGTM

I left some minor comments, but aren't blockers. Just worried about adding dependencies that are not really needed

File: alerta-covid/Gemfile (hunk 1 above)
Comment (ArielMD) [reply alerta-covid/Gemfile:30]: Sorry :c, but we don't understand which gem we can make with ​​vanilla ruby/rails, are marked 4

File: alerta-covid/Gemfile (hunk 2 above)
Comment (ArielMD) [reply alerta-covid/Gemfile:31]: If we have time, we would like to generate the reports xd

File: alerta-covid/Gemfile (hunk 1 above)
Comment (EduardoGHdez) [reply alerta-covid/Gemfile:30]: Figaro. I can see what the purpose of that gem is
> Figaro is inspired by the Twelve-Factor App methodology, which states:

But Rails is already 12-factor friendly, and I think Figaro it's not needed 😄

Comment (ArielMD) [reply alerta-covid/Gemfile:30]: Figaro we use it to use and config enviroment variables, so we dont have to upload our users and passwords in github

**Review by vyk2rr — APPROVED** [review]
LGTM ✅ 

you don't even no need to, but extra points if you add a:
- **Description of the Pull Request**: small description of what you are implementing, and any helpful information presented in a short and quick text, and a
- **Background**: a full description of what is being requested, did you tried other things and they didn't work? any decisions taken beside the defaults (because of any reason; list them here), commands that you used, settings that were changed, and why, like a Technical Spec. What do you think?

😄

**Review by ArielMD — APPROVED** [review]
that looks good, we just need to add the environment variables for the DB

**Review by ramasoftdev — APPROVED** [review]

**Pull request merged**